packages/web/README.md

```diff
@@ -39,6 +39,23 @@ Currently the client only supports the following entrypoints, with the same inte
 - [getInstanceInfo](../nodejs#getInstanceInfo)
 - [getConsensusStatus](../nodejs#getConsensusStatus)
 
+# Detecting the Concordium browser wallet API
+The SDK provides a helper method that can be used to get access to the Concordium browser wallet API. The helper ensures that the browser wallet API is correctly injected before exposing the API methods to the caller. The following exemplifies how accessing the API can be done.
```

Comment:
Should we link to the api/wallet somehow here? :thinking: 
(Though I'm not sure where to link to)